frontend/src/components/LinkGenerator/index.tsx

```diff
@@ -30,8 +42,23 @@ const LinkGenerator: React.FC<CreateTokenProps> = ({
 
   return (
     <div>
+      <h3>Create {role_label} Invite Link</h3>
+      <select
+        value={duration === undefined ? "" : duration}
```

Comment:
If we are defaulting to 7 day expiration, we should probably default to that option in the dropdown.

Reply:
the drop down does default to it.